### ISSUE TITLE
🆕 Software Suggestion | Bludit

### ISSUE THREAD

**Issue by misaka00251** [privacytoolsIO/privacytools.io#1825]
## Basic Information

**Name:** Bludit
**Category:** Productivity Tools
**URL:** https://www.bludit.com/

## Description

Another Wordpress & write.as alternative for personal usage.

Bludit is a web application to build your own website or blog in seconds, it's completely free and open source. Bludit uses files in JSON format to store the content, you don't need to install or configure a database. You only need a web server with PHP support.

## Why I am making the suggestion

It's fast, simple, you can install it without any tech knowledge.

## My connection with the software

No.

- [x] I will keep the issue up-to-date if something I have said changes or I remember a connection with the software.

**Comment by blacklight447-ptio** [comment]
While it seems like a nice little project, we already have quite a bit of recommendations on the site for productivity tools, and a lot of worth mentionings. i tend to think that listing to many may make it harder for people to choose, and its yet another project we need to keep an eye on to see if anything shady happens. @privacytoolsIO/editorial thoughts?

**Comment by danarel** [comment]
I agree with @blacklight447-ptio - I think it looks nice, but with all the recommendations we have, plus this only offering self-hosting, just makes it slightly less attractive than what we have already.

And yes, too many projects does mean too many things we need to monitor.
